.github/actions/setup/action.yaml

```diff
@@ -9,7 +9,7 @@ runs:
       uses: defenseunicorns/setup-zarf@main
       with:
         # renovate: datasource=github-tags depName=defenseunicorns/zarf versioning=semver
-        version: v0.31.1
+        version: v0.32.1
         download-init-package: true
```

Comment:
noting that this action is downloading the `init` package, but is not calling `zarf init`, and `tasks/setup.yaml` is `init`ing via OCI, prob going to cause some confusion down the road